protocol-list.md

```diff
@@ -41,3 +41,10 @@ publications.  The protocol is available in
 [Fujii et al, 2014](https://www.nature.com/articles/nprot.2014.107).
 There is also a [JOVE article](https://www.jove.com/video/55282).
 
+## Spencer et al,JOVE,2013
+The Encapsulation of Cell-free Transcription and Translation Machinery in Vesicles for the Construction of Cellular Mimics | 
```

Comment:
Remove vertical bar at the end of the line.  Also for the other entries we haven't put in the title of the article.  OK to do that here, it is just a slightly different style.